### PR TITLE
Add mechanism to cleanup local stream and websocket

### LINKED ISSUE #6399: How to destroy the adaptor in react native
How can i destroy the adaptor Ant media, because the
pong command still showing on the logs and green dot doesn't disappear on the top right corner, even though I trigger the adaptor.stop(stream_id) and navigate back to other screen.

### PR DESCRIPTION
**Pull request by Void-Spectra**
https://github.com/ant-media/Ant-Media-Server/issues/6399

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -84,6 +86,11 @@ export default function App() {
     console.log('in verify');
     if (adaptor.localStream.current && adaptor.localStream.current.toURL()) {
       console.log('in verify if adaptor local stream', adaptor.localStream);
+      if (isWaitingWebsocketInit) {
+        setIsWaitingWebsocketInit(false);
+        publishStreamId = generateRandomString(12);
```
Comment (Jastc) [inline example/src/App.tsx:105]: What is the idea of this line (91)? Is it possible to remove this line? Without it it works as expected !

File: example/src/App.tsx [hunk 2]
```diff
@@ -84,6 +86,11 @@ export default function App() {
...
+        adaptor.publish(streamNameRef.current);
```
Comment (Jastc) [inline example/src/App.tsx]: Can this not be replaced with `handlePublish()` ?

Comment (Void-Spectra) [reply example/src/App.tsx]: Yes, it's possible. You can make the publish stream id persistant.

File: example/src/App.tsx (hunk 1 above)
Comment (Void-Spectra) [reply example/src/App.tsx:105]: the idea is preventing the stream id already in use issue but if you set it according to your use case, you can use the same stream if all the time.

**Review by lastpeony — APPROVED** [review]
lgtm. tested on my local

**Pull request merged**